### PR TITLE
PLT-6657 Move system console to use v4 endpoints and redux

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Move system console to use v4 endpoints and redux.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-6657

### CODE REVIEW

File: webapp/actions/admin_actions.jsx [hunk 1]
```diff
@@ -2,201 +2,173 @@
...
 import {browserHistory} from 'react-router/es6';
 
 // Redux actions
 import store from 'stores/redux_store.jsx';
 const dispatch = store.dispatch;
 const getState = store.getState;
 
-import {getUser} from 'mattermost-redux/actions/users';
+import {updateUserMfa, updateUserPassword} from 'mattermost-redux/actions/users';
+import {
```
Comment (hmhealey) [inline webapp/actions/admin_actions.jsx]: At some point, should we just go back to having this do `import * as AdminActions from ...`? :p

File: webapp/components/admin_console/brand_image_setting.jsx [hunk 2]
```diff
@@ -37,9 +40,15 @@ export default class BrandImageSetting extends React.Component {
     }
 
     componentWillMount() {
-        $.get(Client.getAdminRoute() + '/get_brand_image?t=' + this.state.brandImageTimestamp).done(() => {
-            this.setState({brandImageExists: true});
-        });
+        fetch(`${Client4.getBrandRoute()}/image?t=${this.state.brandImageTimestamp}`).then(
```
Comment (hmhealey) [inline webapp/components/admin_console/brand_image_setting.jsx]: Can you make a method in the Client4 to construct the URL?

File: webapp/components/admin_console/brand_image_setting.jsx [hunk 3]
```diff
@@ -130,7 +139,7 @@ export default class BrandImageSetting extends React.Component {
             img = (
                 <img
                     className='brand-img'
-                    src={Client.getAdminRoute() + '/get_brand_image?t=' + this.state.brandImageTimestamp}
+                    src={`${Client4.getBrandRoute()}/image?t=${this.state.brandImageTimestamp}`}
```
Comment (hmhealey) [inline webapp/components/admin_console/brand_image_setting.jsx]: And you can use it here

File: webapp/components/admin_console/brand_image_setting.jsx (hunk 2 above)
Comment (jwilander) [inline webapp/components/admin_console/brand_image_setting.jsx]: Is this really needed? It's only used in the two places

File: webapp/actions/admin_actions.jsx (hunk 1 above)
Comment (crspeller) [inline webapp/actions/admin_actions.jsx]: Ya this is a bit ridiculous :P

**Comment by jwilander** [comment]
@hmhealey changes made

**Pull request merged**